libr/bin/format/omf/omf.c

```diff
@@ -751,9 +751,9 @@ int r_bin_omf_send_sections(RList *list, OMF_segment *section, r_bin_omf_obj *ob
 
 		// if index == 0, it's mean there is no name
 		if (section->name_idx && section->name_idx - 1 < obj->nb_name) {
-			snprintf (new->name, R_BIN_SIZEOF_STRINGS, "%s_%d", obj->names[section->name_idx - 1], ct_name++);
+			new->name = r_str_newf ("%s_%d", obj->names[section->name_idx - 1], ct_name++);
```

Comment:
@radare is there a `r_str_nnewf` that would limit the maximum size of the string (e.g. similar to snprintf)? I'm not sure we can assume obj->names[..] is null terminated. can we?